content/en/docs/contributing/blog.md

```diff
@@ -27,15 +39,28 @@ Unsuitable content includes:
 
 - Vendor product pitches
 
-To submit a blog post,
+If your blog post fits into the list of appropriate content,
 [raise an issue](https://github.com/open-telemetry/opentelemetry.io/issues/new?title=New%20Blog%20Post:%20%3Ctitle%3E)
-with the title and a short description of your blog post. If you are not a
-[Member](https://github.com/open-telemetry/community/blob/main/community-membership.md#member),
-you also need to provide a _sponsor_ for your blog post, who is a Member (by
-that definition) and who is willing to provide a first review of your blog post.
-
-If you do not raise an issue before providing your PR, we may request you to do
-so before providing a review.
+with the following details:
+
+- Title of the blog post
+- Short description and outline of your blog post
+- If applicable, list the technologies used in your blog post. Make sure that
+  all of them are open source, and prefer CNCF projects over non-CNCF projects
+  (e.g. use Jaeger for trace visualization, and Prometheus for metric
+  visualization)
+- Name of a [SIG](https://github.com/open-telemetry/community/), which is
+  related to this blog post
+- Name of a sponsor (maintainer or approver) from this SIG, who will help to
```

Comment:
The previous step requires that a SIG be identified, which is a great step forward. I'd suggest that they cc the SIG approvers in the issue. We docs-approvers can then ensure that the corresponding SIG approvers are listed as require approvers for the PR.

So, I might be tempted to suggest that **finding a sponsor be _optional_** (for now), but state that having a sponsor _may_ increase their chances of having their blog reviewed and approved more quickly (or some such language). WDYT?

Reply:
I agree, keeping it optional for now makes our life easier as well for a start, since we do not have to push back that hard (based on our policy) but can let people know that we see their PR but it'S not our prio right now and they should get a sponsor to help themselves